src/macaron/repo_finder/commit_finder.py

```diff
@@ -276,26 +271,20 @@ def find_commit_from_version_and_name(git_obj: Git, name: str, version: str) ->
         # Tag names are taken from valid_tags and should always exist within it.
         logger.debug("Missing tag name from tag dict: %s not in %s", tag_name, valid_tags.keys())
 
-    branch_name = _get_branch_of_commit(git_obj.get_commit_from_tag(tag_name))
     try:
         hexsha = tag.commit.hexsha
     except ValueError:
         logger.debug("Error trying to retrieve digest of commit: %s", tag.commit)
-        return "", ""
-
-    if not branch_name:
-        logger.debug("No valid branch associated with tag (commit): %s (%s)", tag_name, hexsha)
-        return "", ""
+        return ""
 
     logger.debug(
-        "Found tag %s with commit %s of branch %s for artifact version %s@%s",
+        "Found tag %s with commit %s for artifact version %s@%s",
         tag,
         hexsha,
-        branch_name,
         name,
         version,
     )
-    return branch_name, hexsha
+    return str(hexsha)
```

Comment:
Same comment. Can `hexsha` be `None`?

Reply:
The above applies here too as the underlying class is the same.

Reply:
So, we don't need to create a `str` object, right?